src/magento-payments/payoutreports.md

```diff
@@ -0,0 +1,133 @@
+---
+ee_only: false
+title: Payout Reports
+---
+
+Payout reports show comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+You can efficiently cross-reference orders and payments and reconcile accounts---all from one place.
```

Comment:
The user can efficiently cross-reference because the payments and reconcile accounts are in the same place. Would it be helpful to explain what makes it efficient?

Reply:
Yes, especially given our audience. Good reminder.